changelog.txt

```diff
@@ -1,3 +1,8 @@
+Version 2.0.15
+-----------
+- Introduces additional tests for cache resiliency
+- Fixes an encoding issue faced by devices configured for Turkish locale
```

Comment:
nit: #963

Reply:
Thanks; resolved in fd5f2f8cc525b95e46782624add661b0e7d1cec2